data/json/professions.json

```diff
@@ -5137,6 +5137,146 @@
                 "bra",
                 "panties"
             ]
+        },
+            {
```

Comment:
Right now `escape_con` is nested under the Tourist profession. This line needs to have another closing bracket. Have you played on this PR?

Currently it will throw this error:
<img width="1018" alt="screen shot 2017-09-04 at 12 53 21 pm" src="https://user-images.githubusercontent.com/714017/30034512-0e01b6a4-9170-11e7-9960-a5a1b68a612b.png">

Reply:
Fixed, I did't remove anything so I don't understand why that happened.